test/test-readability.js

```diff
@@ -60,7 +60,7 @@ describe("Test page", function() {
         host: "fakehost",
         prePath: "http://fakehost",
         scheme: "http",
-        pathBase: "http://fakehost/test"
+        pathBase: "http://fakehost/test/"
```

Comment:
Note: it seems that's always what's returned by Gecko `newURI().spec`, though I'm not 100% sure; can someone confirm?